Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Modifications for Solaris build.

--- imap-2007f/src/osdep/unix/Makefile_orig	Fri Jul 22 17:20:10 2011
+++ imap-2007f/src/osdep/unix/Makefile	Mon Nov  7 13:37:37 2011
@@ -54,7 +54,7 @@
 
 # Extended flags needed for SSL.  You may need to modify.
 
-SSLDIR=/usr/local/ssl
+SSLDIR=/usr
 SSLCERTS=$(SSLDIR)/certs
 SSLKEYS=$(SSLCERTS)
 SSLINCLUDE=$(SSLDIR)/include
@@ -724,9 +724,9 @@
 	 SPOOLDIR=/var/spool MAILSPOOL=/var/mail \
 	 ACTIVEFILE=/usr/share/news/active \
 	 RSHPATH=/usr/bin/rsh \
-	 BASECFLAGS="-g -O -w" \
+	 BASECFLAGS="$$CFLAGS" \
 	 BASELDFLAGS="-lsocket -lnsl -lgen" \
-	 RANLIB=true CC=/opt/SUNWspro/bin/cc
+	 RANLIB=true CC="$$CC"
 
 
 # Note: It is a long and disgusting story about why cc is set to ucbcc.  You
